certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/ActionRuleChildRelationNotSigned.java

```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.certificateservice.domain.action.model;
+
+import java.util.List;
+import java.util.Optional;
+import se.inera.intyg.certificateservice.domain.certificate.model.Certificate;
+import se.inera.intyg.certificateservice.domain.certificate.model.Relation;
+import se.inera.intyg.certificateservice.domain.certificate.model.Status;
+
+public class ActionRuleChildRelationNotSigned implements ActionRule {
+
+  @Override
+  public boolean evaluate(Optional<Certificate> certificate,
+      Optional<ActionEvaluation> actionEvaluation) {
+    return certificate
+        .map(Certificate::children)
+        .filter(this::notSigned)
```

Comment:
Is it the same rule no matter what type of relation? Thinking about COMPLEMENT (should never happen) and RENEW?

Reply:
Did not consider the effect this will have for other child relations. I verified that this rule does not apply for RENEW, I will make the required updates while considering the comment about utilizing ActionRuleChildRelationNoMatch.